### ISSUE TITLE
Text bubble render error on second line with special character

### ISSUE THREAD

**Issue by jeifour** [ChatSecure/ChatSecure-iOS#1081]
Hi, I just found out when I write a message and it has multiple lines, that the second line will be blank when I include a special character, in my case I was trying to send a ❤︎.

Not sure if this is also the case for other characters or why it works fine in the first line. You can see the whole message I sent correctly on the contact/chat list, so it‘s probably only a display error.

**Comment by mimi89999** [comment]
@jeifour I can't reproduce this issue. All characters appear when there is a newline and when there is a line wrap. Could you please confirm that the issue has been resolved?

**Comment by jeifour** [comment]
@mimi89999 Thanks for your reply. I am able to reproduce this. I have the newest version from the App Store 4.3.6 (138) on my iPhone SE with iOS 12.1.4 (16D57).

<img width="320" alt="png 2" src="https://user-images.githubusercontent.com/3247063/53525772-d7db3c80-3ae2-11e9-81f9-155b45b9315f.png">
<img width="320" alt="png" src="https://user-images.githubusercontent.com/3247063/53525773-d873d300-3ae2-11e9-85f3-63850f2f4746.png">

The second message in my screenshot really included this type of heart at the end: ❤︎ Which is visible fine on the contact screen.

**Comment by mimi89999** [comment]
I tried sending the same message on a simulated iPhone SE running iOS 12.1, but it appeared correctly. Here is the screenshot:

![simulator screen shot - iphone se - 2019-03-03 at 18 57 01](https://user-images.githubusercontent.com/8530546/53699267-b205ae80-3de6-11e9-9d91-96c809730908.png)

I don't know how to debug this.

**Comment by mimi89999** [comment]
OK. I managed to reproduce it. The problematic symbol isn't the heart:
> The second message in my screenshot really included this type of heart at the end: ❤︎

It's the `U+2764` _"Heavy Black Heart"_ symbol.

**Comment by mimi89999** [comment]
It's an issue in a library ChatSecure is using: https://github.com/jessesquires/JSQMessagesViewController/issues/1852

**Comment by mimi89999** [comment]
I tested with many different characters and only that one was causing the issue.

**Comment by jeifour** [comment]
Yes sorry the "emoji heart" I only used to show an example where it works. I now see that my explanation wasn't clear enough. Glad you still managed to reproduce it.
